mirdata/datasets/salami.py

```diff
@@ -158,20 +158,30 @@ def genre(self):
         return self._track_metadata.get("genre")
 
     @core.cached_property
-    def sections_annotator_1_uppercase(self) -> Optional[annotations.SectionData]:
-        return load_sections(self.sections_annotator1_uppercase_path)
-
-    @core.cached_property
-    def sections_annotator_1_lowercase(self) -> Optional[annotations.SectionData]:
-        return load_sections(self.sections_annotator1_lowercase_path)
-
-    @core.cached_property
-    def sections_annotator_2_uppercase(self) -> Optional[annotations.SectionData]:
-        return load_sections(self.sections_annotator2_uppercase_path)
+    def sections_uppercase(self) -> Optional[annotations.MultiAnnotator]:
```

Comment:
to make this a non-breaking change, we can deprecate it by keeping the existing attributes `sections_anntotator_1_uppercase` as they are but logging a deprecation warning, and pointing to the new attribute they should use:

something like:
```def sections_anntotator_1_uppercase(self):
    logging.warning("Deprecation warning: sections_anntotator_1_uppercase is deprecated starting in version <whatever version we are now> and will be removed in a future version. Use sections_uppercase in the future.
```

Reply:
It's true, though I suspect that the `salami` loader doesn't have many users since the audio is private XD, so I'm fine with the breaking change actually. Thoughts?

Reply:
so far in the other loaders I've been doing the deprecation thing, and I think it's good practice. You don't have to change any of your current code, just copy back the existing attributes and add the deprecation warnings.